Lib/ufo2ft/kernFeatureWriter.py

```diff
@@ -1,6 +1,8 @@
 from __future__ import print_function, division, absolute_import, unicode_literals
+from fontTools.misc.py23 import unichr
 
 import re
+import unicodedata
```

Comment:
it'd be nice if you could try to `import unicodedata2` (the py3 backport) and use that if it's available.